NEWS.md

```diff
@@ -33,15 +33,58 @@ The format is based on [Keep a Changelog]
 
 ## [unreleased]
 
+The most noteworthy additions in this release are:
+
+1. Titlebar window icons and layout configuration
+2. Support for the cosmic-workspace protocol and the openbox inspired
+   client-list-combined-menu for a better user experience with workspaces.
+
+Notes to package maintainers:
+
+- The SSD titlebar window icon support requires libsfdo to be added as a
+  dependency or statically linked. If this is not wanted, add -Dicon=disabled to
+  the `meson setup` command in the build script for the next release.
+- PRs #1716 and #2205 add labwc xdg-portal configuration, modify `labwc.desktop`
+  and amend `XDG_CURRENT_DESKTOP` which should enable better out-of-the-box
+  support for xdg-desktop-portal, but if you already ship a custom setup for
+  this you or have different requirements, please review this change.
+
 ### Added
 
-- Add theme options for circular button hover effect, button padding and button
-  spacing. Written-by: @jp7677 (#2127)
+- Support dmabuf feedback (#2234, #1278)
+- Add initial implementation of cosmic-workspace-unstable-v1 (#2030)
+- Optionally support SSD titlebar window icons. When an icon file is not found
+  or could not be loaded, the window menu icon is shown as before.  The icon
+  theme can be selected with `<theme><icon>` (#2128)
+- Add action `UnSnap`. This behaves like `ToggleSnapToEdge/Region` but
```

Comment:
Since this line refers to `ToggleSnapToEdge/Region` we should mention this after the next point, which introduces `ToggleSnapToEdge/Region`.